tutorials/beaconkit.md

```diff
@@ -0,0 +1,113 @@
+# Building a Sovereign Rollup with BeaconKit and Rollkit
+
+![beaconkit](https://camo.githubusercontent.com/8aaae79e171969a2a9c950582d512cd1e3746e67d3aea6410afc04e9b6cb8055/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f6475763067343032792f696d6167652f75706c6f61642f76313731383033343331322f426561636f6e4b697442616e6e65722e706e67)
+
+## Introduction
+
+This tutorial guides developers through the process of building and running a sovereign rollup using BeaconKit and Rollkit. BeaconKit is a modular framework for building EVM-based consensus clients, while Rollkit is a modular framework for building sovereign rollups. By combining these tools, you can create your own customizable rollup chain using the Ethereum Virtual Machine (EVM).
+
+## Prerequisites
+
+Before starting, ensure you have the following software installed on your machine:
+
+* [Rollkit](https://rollkit.dev/): A modular framework for sovereign rollups
+* [Docker](https://www.docker.com/): For running containerized applications
+* [Go](https://golang.org/) (version >= 1.21.0): The programming language used by BeaconKit
+* [jq](https://stedolan.github.io/jq/): A lightweight command-line JSON processor
+
+## Run a local Data Availability (DA) node {#run-local-da-node}
+
+First, set up a local data availability network node:
+
+```bash
+cd $HOME && curl -sSL https://rollkit.dev/install-local-da.sh | sh -s {{constants.localDALatestTag}}
+```
+
+This script builds and runs a DA node, which will listen on port `7980`.
+
+## Clone the BeaconKit repository {#clone-the-repo}
+
+Clone the BeaconKit repository and switch to the Rollkit branch:
+
+```bash
+cd $HOME
+git clone -b rollkit https://github.com/rollkit/beacon-kit.git
+cd beacon-kit && git checkout rollkit
+```
+
+## Start the Go Ethereum (Geth) client {#start-go-ethereum-client}
+
+Start a Go Ethereum client to provide the Ethereum layer for your rollup:
+
+```bash
+cd $HOME/beacon-kit
+make start-geth
+```
+
+This command initializes a new Ethereum node with a specified genesis file and starts the Geth node with specific configuration options.
```

Comment:
Ethereum node? Should be renamed to Rollkit node right?

Reply:
ethereum node